app/scripts/directives/editEnvironmentFrom.js

```diff
@@ -27,6 +29,16 @@
 
     ctrl.setFocusClass = 'edit-environment-from-set-focus-' + uniqueId;
 
+    ctrl.viewOverlayPanel = function(entry) {
+      ctrl.decodedSecretData = SecretsService.decodeSecretData(entry.data);
```

Comment:
We shouldn't need to call this if we're not showing the secret content. For now, I'd just use `entry.data` and not decode.